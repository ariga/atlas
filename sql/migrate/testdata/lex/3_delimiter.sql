-- atlas:delimiter \n\n

CREATE INDEX i1 ON t1(c1)

CREATE INDEX i2 ON t2(c2)