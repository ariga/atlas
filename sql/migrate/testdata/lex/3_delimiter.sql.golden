CREATE INDEX i1 ON t1(c1)
-- end --
CREATE INDEX i2 ON t2(c2)